src/framework/ui/view/iconcodes.h

```diff
@@ -291,6 +291,8 @@ class IconCode
         NOTE_DOTTED_4 = 0xF397,
 
         COPY = 0xF398,
+        CUT = 0xF39A,
+        PASTE = 0xF399,
```

Comment:
Could you swap those two, to have them in numeric order?

Reply:
Sure!

Reply:
Thanks